src/components/structures/TimelinePanel.js

```diff
@@ -1023,7 +1020,14 @@ var TimelinePanel = React.createClass({
             var boundingRect = node.getBoundingClientRect();
             if ((allowPartial && boundingRect.top < wrapperRect.bottom) ||
                 (!allowPartial && boundingRect.bottom < wrapperRect.bottom)) {
-                return i;
+                let latestReadEventIndex = i;
+                // Place the RM at a hidden event below the latest seen event (if exists)
```

Comment:
Actually this is confusing, the event is just the latest read visible event... `event (if exists)` should be probably be `visible event`